readme.md

```diff
@@ -213,6 +213,7 @@
 - [bugger](https://github.com/buggerjs/bugger) - Provides Chrome Devtools bindings to debug programs in Chrome.
 - [0x](https://github.com/davidmarkclements/0x) - Flamegraph profiling.
 - [ctrace](https://github.com/automation-stack/ctrace) - Well-formatted and improved trace system calls and signals.
+- [Vorlon.JS](http://www.vorlonjs.com/) - Open source tool for remotely debugging and testing your JavaScript.
```

Comment:
The words `Open source tool` and `JavaScript` is moot as those are implied.

I would rather link to http://www.vorlonjs.com/documentation/#debugging-nodejs-applications It's pretty unclear from the homepage that the tool works for debugging Node.js apps too.